Drop-In/src/main/java/com/braintreepayments/api/DropInViewModel.java

```diff
@@ -0,0 +1,39 @@
+package com.braintreepayments.api;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import java.util.List;
+
+public class DropInViewModel extends ViewModel {
+
+    private final MutableLiveData<Boolean> isLoading = new MutableLiveData<>();
+
+    private final MutableLiveData<List<DropInPaymentMethodType>> supportedPaymentMethods = new MutableLiveData<>();
+    private final MutableLiveData<List<PaymentMethodNonce>> vaultedPaymentMethods = new MutableLiveData<>();
+
+    public LiveData<Boolean> isLoading() {
+        return isLoading;
+    }
+
+    void setIsLoading(boolean value) {
+        isLoading.setValue(value);
+    }
+
+    void setVaultedPaymentMethods(List<PaymentMethodNonce> value) {
+        vaultedPaymentMethods.setValue(value);
+    }
+
+    public LiveData<List<PaymentMethodNonce>> getVaultedPaymentMethods() {
+        return vaultedPaymentMethods;
+    }
+
+    public void setSupportedPaymentMethods(List<DropInPaymentMethodType> supportedPaymentMethods) {
```

Comment:
Nitpick but let's go with `value` for all of the setter variables or rename them all to align with the type names.

Reply:
Def!